index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset = "UTF-8">
+    <meta name = "viewport" content = "width=device-width, initial-scale=1.0">
+    <title>Youtube</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav class = "headerBar">
+            <div class = "logos"> 
+                <a href = "#"><img class = "logoMenu" src = "images/menu-icon.PNG" alt = "Youtube Logo" ></a>
+                <a href = "#"><img class = "logoYoutube" src = "images/logo-icon.PNG" alt = "Youtube Logo"></a>
+            </div>
+            <div class = "search">
+                <input id = "searchBar" type = "text" placeholder = "Search">
+                <button id = "searchButton"><img src = "images/search-icon.png" alt = "Search Icon"></button>
+            </div>
+            <div class = "icons">
+                <img id = "icon1" src = "images/camera-icon.PNG" alt = "Camera Icon" width = "28" height = "20">
+                <img id = "icon2" src = "images/menu-icons.PNG" alt = "Menu Icon" width = "25" height = "20">
+                <img id = "icon3" src = "images/bell-icon.PNG" alt = "Bell Icon" width = "20" height = "20">
+                <img id = "icon4" src = "images/user-icon.PNG" alt = "User Icon" width = "38" height = "33">
+            </div>
+        </nav>
```

Comment:
Inline styling is considered bad practise. Kindly let's remove all the inline styling in our markup and style our HTML file using stylesheets only thanks.

Reply:
## UPDATE: CORRECTION ON THE HIGHLIGHTED REQUESTED CHANGES.
- I noticed and with a great eye-help from my colleague @adaorachi, I must have slipped a bit in requesting for these changes in the manner in which I did. It was not intentional and I am sorry if this could have misled you in any way. 

### Please consider these changes optional.
- If we are to consider removing the Width and Height HTML image attributes from our markup, it would look great and make our markup appear a bit cleaner. I'd suggest we use CSS for the styling in this section instead. Thank you 😄

Reply:
Hi Christian, don't worry its awesome to have recommendations from people with a great experience like you! We did the changes that you suggested and sent our code to review again. 😄